### ISSUE TITLE
Takeover linking to equations from MathJax

### ISSUE THREAD

**Issue by rbeezer** [rbeezer/mathbook#1354]
We use a semi-elaborate mechanism provided by MathJax to link in HTML output to specific equations of a display.  But when we rip out math for offline processing and placement in EPUB, braille, and speech, this linking is lost.

We can take a hint from knowl creation and just link to an enclosing container of the display.  But maybe wait for MathJax 3 (#1039).

Some relevant code comments.  See also 44728ecfe084b9cba1fb9741f37670e9552a470f

```
2020-08-17: \label{} gets butchered in the SRE conversion     
to Nemeth braille, so we are using stripped down versions     
of these templates in the "extract-math" stylesheet (look     
there before editing here).  Presently, the useLabelIDs       
scheme with the "mjx-eqn-foo" manufactured is used in HTML    
output.  But maybe we should mimic knowl creation and have    
a cross-reference (in-context) point to an ID on an           
(existing) enclosing "div" on display mathematics.  That      
might be better for EPUB, speech, etc, in addition to braille.
```

```
We want the hard-coded "tags" (numbered equations) that PreTeXt
supplies, but we do not want the "\label{}" that comes along as
part of MathJax's semi-automatic linking mechanism.  Since we  
disassociate the math from its surroundings, any sort of       
cross-referencing of equations should be handled another way.  
But we do need to "name" equations.                            
                                                               
These are stripped-down versions from the (imported) -html     
conversion.  It is possible they will become identical some    
day and then these versions can go away.                       
```

**Comment by rbeezer** [comment]
See #1473

**Comment by pkra** [comment]
I use the [\cssId{id}{} macro](http://docs.mathjax.org/en/latest/input/tex/extensions/html.html?highlight=cssid#html) to place IDs in equations when they're generated "elsewhere" (and disable MathJax's own ID generation which can generate invalid IDs, depending on the label content).

**Comment by rbeezer** [comment]
Aah, thanks @pkra.  I didn't know about that one.  I think we do a pretty rigorous check on crazy labels, but this might be a better mechanism if we don't use outr own enclosing `div.displaymath`.

**Comment by rbeezer** [comment]
Done, at 6aa3d689a9980b45c028df7201549e546540fb3d, plus one or two following.

The `\css{id}{}` macro might not solve the problem of an `xref` across pages where the math is not generated in time for the fragment identifier to be effective.
